clear-containers-selinux/clear-containers-selinux.spec

```diff
@@ -21,7 +21,7 @@
 # Package information
 Name:			clear-containers-selinux
 Version:		0.1
-Release:		1%{?dist}
+Release:		5.<B_CNT>
```

Comment:
I am not familiar with this what it represents ?

Reply:
<B_CNT> is needed in order to have version bumps, so new packages of the same version can be published to the download server. Without this, OBS will never publish a package that has been bumped.

Reply:
It actually makes the `Release` field work as expected in OBS.